ansible/roles/jupyterhub/templates/env.service.j2

```diff
@@ -1,8 +1,11 @@
 # api token should be equal to value set for jupyterhub
 JUPYTERHUB_API_TOKEN={{jhub_api_token}}
 
+# external facing api url
+JUPYTERHUB_BASE_URL={{base_url}}
```

Comment:
is this new variable needed here?

Reply:
For not it's not really needed. I'll remove it and only leave this value in the jupyterhub and setup course envs.